pkg/operator/ceph/object/objectstore.go

```diff
@@ -389,55 +389,61 @@ func getZoneEndpoints(objContext *Context, serviceEndpoint string) ([]string, bo
 	return zoneEndpointsList, isEndpointAlreadyExists, nil
 }
 
+func createMultisiteConfigurations(objContext *Context, configType, configTypeArg string, args ...string) error {
```

Comment:
Is there a unit test we can add to make sure this regression doesn't happen again?

Reply:
@parth-gr Is this just a refactor? Or what bug is being fixed by this new method?

Reply:
@travisn if you remember the discussion this was the original error/bug https://github.com/rook/rook/pull/12817#discussion_r1313364512

Reply:
@BlaineEXE added unit test